exist-core/src/main/java/org/exist/util/FastStringBuffer.java

```diff
@@ -471,12 +472,12 @@ public String getNormalizedString( int mode ) {
      *@param  sb    Description of the Parameter
      *@param  mode  Description of the Parameter
      *@return       The normalizedString value
-     */    
+     */
     public StringBuffer getNormalizedString(StringBuffer sb, int mode ) {
```

Comment:
Should this be changed to use `StringBuilder` rather than String buffer?

Reply:
the classname is Fast...StringBuffer :-) but it makes sense in a way, I will check the users of the method.